pcmanfm-qt.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+   <id>io.github.lxqt.pcmanfm-qt</id>
+   <metadata_license>CC-BY-4.0</metadata_license>
+   <project_license>GPLv2</project_license>
+   <name>PCManFM-Qt</name>
+   <summary>Filemanager</summary>
+   <description>
+ <p>
+    Fast filemanager and desktop manager written in Qt. Among
+    its features are:
+    </p>
+    <ul>
+      <li>Split view</li>
+      <li>Filter items by typing</li>
+      <li>Full wayland compatible</li>
+      <li>Display options per folder</li>
+      <li>Connect to server</li>
+      <li>Admin mode fot tabs</li>
+      <li></li>
+      <li></li>
+      <li></li>
+    </ul>
+   </description>
+   <launchable type="desktop-id">pcmanfm-qt.desktop</launchable>
```

Comment:
Not strictly required because `<anything>.desktop` is still supported as a legacy value, but when converting component IDs to reverse-DNS format, [they suggest](https://www.freedesktop.org/software/appstream/docs/chap-Validation.html#asv-cid-desktopapp-is-not-rdns):

> You may also consider to update the name of the accompanying .desktop file to follow the latest version of the Desktop-Entry specification and use a rDNS name for it as well. In any case, do not forget to mention the new desktop-entry in a `<launchable/>` tag for this component to keep the application launchable from software centers and the .desktop file data associated with the metainfo data.

So the `.desktop` file could ideally be renamed to `io.github.lxqt.pcmanfm-qt.desktop`, to follow the latest FreeDesktop Desktop-Entry spec (and _also_ be more in line with appstream).